docs/developer-guide.md

```diff
@@ -1,8 +1,115 @@
 # Developer Guidelines
 
-## Setting up the commit-msg hook for the pre-commit
+## Setting up the Local Development Environment
 
-Run `pre-commit install --hook-type commit-msg` to install the commit-msg hook.
+### Prerequisites
+
+Ensure the following are installed:
+
+- [Postgres](https://www.postgresql.org/)
+- [Sqitch](https://sqitch.org/) (requires a working Perl 5 installation)
+- [Node](https://nodejs.org/)
+- [asdf](https://asdf-vm.com/#/core-manage-asdf) (using your system's package manager & follow instructions to add it to your shell)
+
```

Comment:
These will all be installed by the `make install_dev_tools` target if not present